scripts/circle-ci/run-circle-tests.sh

```diff
@@ -77,6 +77,8 @@ JAVA_GC_LOGGING_OPTIONS="${JAVA_GC_LOGGING_OPTIONS} -Xlog:gc:build-%t-%p.gc.log"
 # External builds have a 16gb limit.
 if [ "$test_suite_index" -eq "14" ]; then
     export _JAVA_OPTIONS="-Xms2g -Xmx4g -XX:ActiveProcessorCount=8 ${JAVA_GC_LOGGING_OPTIONS}"
+elif [ "$test_suite_index" -eq "4" ]; then
+    export _JAVA_OPTIONS="-Xms8g -Xmx8g -XX:ActiveProcessorCount=8 ${JAVA_GC_LOGGING_OPTIONS}"
```

Comment:
Needed as test suite 4 is GC'ing super hard, which is causes flakes occasionally, even in develop